### ISSUE TITLE
Apps array not being added to profile when logging into apps if using a custom gaia hub

### ISSUE THREAD

**Issue by jehunter5811** [blockstack/blockstack-browser#1806]
I created a name (`graphitehunter.id.blockstack`) using the CLI. I created that ID and pointed it to a locally-hosted custom Gaia hub. Everything seems to work fine as I have an ID, I have a profile.json file, I can log into apps and store data, etc. However, I do not get an updated `apps` array in my profile. See here:

https://core.blockstack.org/v1/search?query=graphitehunter

Or here:

https://s3.us-east-2.amazonaws.com/graphite-docs/1MEG2aCBFRTbhCiAsnrmuAVvx5TUhoWQhS/profile.json

This results in an inability for other users to look up files from this username. Any idea what’s going wrong here?

**Comment by jehunter5811** [comment]
Spoke with Jude and this appears to be because I registered the name and connected to a custom hub via the CLI. I needed to set the browser's API settings to match my local gaia hub URL. 

From Jude: 

```
it looks like a recent change to the profile structure added the `api` section so your profile remembers where the write endpoint for your gaia hub is.  The CLI profile-generation code hasn't yet been updated to add this feature, which probably is what caused the browser to assume that your gaia hub was whatever one was listed in the API settings
```

**Comment by hstove** [comment]
Yeah, what Jude said. All the stuff that updates the profile.json is based on settings in the Browser. You can change your Gaia URL in the 'settings' section of the Browser. After that, I think that the profile should be updated properly if you login again.

**Comment by GInxh** [comment]
I will make sure to have this clarified in the instructions for setting up gaia hubs for develop branch. Other changes that need to happen to enable custom gaia hubs, outside of auto launching images or configuring dev branch.

**Comment by moxiegirl** [comment]
@jcnelson Do we need an issue on the CLI to do this update?

> 

> The CLI profile-generation code hasn't yet been updated to add this feature,

**Comment by jcnelson** [comment]
https://github.com/blockstack/cli-blockstack/issues/13

**Comment by markmhx** [comment]
@GInxh We touched on this during our backlog review today and it appears to be an iceboxed issue – are you actually working on it right now though?
